optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -290,6 +291,10 @@ def __call__(self, trial):
             )
         )
 
+        trial.set_user_attr("elapsed_secs", elapsed_secs)
+        trial.set_user_attr("average_iteration_time", average_iteration_time)
```

Comment:
Just wondering but are the above two attributes used somewhere, or are they there for the user? If so, how about documenting them?

Reply:
That makes sense. I added the description about them in `LightGBMTuner` docstring. To add the `LightGBMTuner` to the document,  the dependency checking in `optuna.integration.lightgbm` seems to be required. So, I'd like to separate the PRs to add the `LightGBMTuner` entry in the reference.

![image](https://user-images.githubusercontent.com/3255979/77732763-e7f87f00-7048-11ea-8516-98f6c4f2347e.png)